Examples/SqlExample.py

```diff
@@ -74,14 +75,14 @@ def CreateDb(self, path):
         connection.execute(CREATE_STMT)
         connection.commit()
 
-        start = time.clock()
+        # start = time.perf_counter()
         i = 0
         while i < NUMBER_OF_ROWS:
             for x in ExampleModel.GetTracks():
                 connection.execute(INSERT_STMT, [i, x.title + str(i), x.artist, x.album, x.sizeInBytes, x.rating])
                 i += 1
         connection.commit()
-        #print "Building database of %d rows took %2f seconds." % (NUMBER_OF_ROWS, time.clock() - start)
+        # print("Building database of %d rows took %2f seconds." % (NUMBER_OF_ROWS, time.perf_counter() - start))
```

Comment:
If we aren't going to track the time here, we can just remove these lines rather than commenting them out

Reply:
The same here: I noticed, that the time is not tracked, so I commented this line out and forgot to remove it.